greynoise/info.json

```diff
@@ -0,0 +1,305 @@
+{
+    "name": "greynoise",
+    "version": "2.0.0",
+    "label": "GreyNoise",
+    "description": "GreyNoise provides information on devices observed mass-scanning the internet. This integration provides a set of actions to lookup IPs or query the GreyNoise API. For support or to report issues or enhancements, please contact support@greynoise.io.",
+    "publisher": "GreyNoise",
+    "icon_small_name": "gn_icon_normal_small.png",
+    "icon_large_name": "gn_icon_normal_small.png",
+    "category": "",
+    "configuration": {
+        "fields": [
+            {
+                "title": "GreyNoise API Key Type",
```

Comment:
Suggestion: Since all inputs in the UI are in shown in the context of the GreyNoise connector itself, the input fields in config and actions can skip the connector name. Eg, this input can be only "API Key Type"